cse-lib/network-processing/src/test/java/com/farao_community/farao/cse/network_processing/TestUtils.java

```diff
@@ -0,0 +1,51 @@
+package com.farao_community.farao.cse.network_processing;
+
+import com.powsybl.commons.datasource.MemDataSource;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.ucte.converter.UcteExporter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public final class TestUtils {
+
+    private TestUtils() {
+
+    }
+
+    public static void assertNetworksAreEqual(Network network, String reference) {
+        MemDataSource dataSource = new MemDataSource();
+
+        UcteExporter exporter = new UcteExporter();
+        exporter.export(network, new Properties(), dataSource);
+
+        try (InputStream actual = dataSource.newInputStream(null, "uct");
+             InputStream expected = TestUtils.class.getResourceAsStream(reference)) {
```

Comment:
You can pass a class in arguments (Class<?> clazz) so that it will not be based on TestUtils that could be placed anywhere

Reply:
ok